action.yml

```diff
@@ -53,20 +53,44 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Check if review is requested
+      shell: bash
+      run: |
+        if [[ "${{ github.event.pull_request.title }}" =~ ^(no|skip)(-|\s)?review|skip(-|\s)?code(-|\s)?review|otter(ai)?(-|\s)?skip|otter(-|\s)?restricted ]]; then
+          echo "🦦 No review requested, skipping code review"
+          echo "code_review_requested=false" >> $GITHUB_OUTPUT
+        else
+          echo "🔍 Code review requested"
+          echo "code_review_requested=true" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Check if PR is merged
```

Comment:
🔍 Similar to the previous step, consider adding a comment to clarify the purpose of checking if the PR is merged.